README.md

```
@@ -109,4 +115,7 @@ If you would like a feature or want to report a bug, please file an issue and we
 [logback-appenders]: https://github.com/y-scope/logback-appenders
 [re2]: https://github.com/google/re2
 [uber-blog]: https://www.uber.com/en-US/blog/reducing-logging-cost-by-two-orders-of-magnitude-using-clp
+[yscope-community-discord]: https://discord.gg/7kZA2m5G87
+[yscope-community-zulip]: https://yscope-clp.zulipchat.com
 [zulip]: https://yscope-clp.zulipchat.com/
```

markdownlint-cli2 (0.17.2)

Link and image reference definitions should be needed
Unused link or image reference definition: "zulip"

(MD053, link-image-reference-definitions)